coderd/agentapi/api.go

```diff
@@ -75,10 +78,6 @@ type Options struct {
 	ExternalAuthConfigs       []*externalauth.Config
 	Experiments               codersdk.Experiments
 
-	// Optional:
-	// WorkspaceID avoids a future lookup to find the workspace ID by setting
-	// the cache in advance.
-	WorkspaceID          uuid.UUID
```

Comment:
This doesn't need to be optional anymore, presumably since we removed the v1 agent api.